Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/HostingOptions.cs

```diff
@@ -12,4 +13,9 @@ public class HostingOptions
     /// back to JSON to return to Lambda.
     /// </summary>
     public ILambdaSerializer Serializer { get; set; }
+
+    /// <summary>
+    /// The encoding response options used by the Lambda
+    /// </summary>
+    public IEncodingOptions EncodingOptions { get; set; }
```

Comment:
`IEncodingOptions` is defined in an `Internal` namespace so it shouldn't be exposed on the public API.